binary_search_tree/binary_search_tree.py

```diff
@@ -5,10 +5,40 @@ def __init__(self, value):
     self.right = None
 
```

Comment:
Great job on BST! I'd like to suggest you put comments with your code , not only will this benefit you but your future employers will appreciate it!